LayoutTests/accessibility/aria-combobox-no-owns.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+<script src="../resources/js-test-pre.js"></script>
+<script src="../resources/accessibility-helper.js"></script>
+</head>
+<body id="body">
+
+<input type="text" role="combobox" id="combobox" aria-label="Combobox">
+<div role="list" id="list" aria-label="List1">
+  <div role="listitem" id="item1">item1</div>
+  <div role="listitem" id="item2">item2</div>
+</div>
+
+<p id="description"></p>
+<div id="console"></div>
+
+<script>
+    description("A combobox should still support aria-activedescendant even if it doesn't use aria-owns.");
+
+    if (window.accessibilityController) {
+        window.jsTestIsAsync = true;
+        var selectedChildrenChangeCount = 0;
+
+        window.accessibilityController.addNotificationListener(function (target, notification) {
+            if (notification == "AXSelectedChildrenChanged" || notification == "AXSelectedRowsChanged") {
+                selectedChildrenChangeCount++;
+                var targetString = platformValueForW3CName(target);
+                debug("Received " + notification + " for " + targetString);
+            }
+        });
+
+        document.getElementById("combobox").focus();
+        var list = accessibilityController.accessibleElementById("list");
+        shouldBe("list.selectedChildrenCount", "0");
+        document.getElementById("combobox").setAttribute("aria-activedescendant", "item1");
+        var listitem1 = accessibilityController.accessibleElementById("item1");
+        setTimeout(async function() {
+            debug("list has " + list.selectedChildrenCount + " selected children");
```

Comment:
We can use `expectAsync` here to ensure we wait for the AX tree to be updated after the DOM change:

```
output += await expectAsync("list.selectedChildrenCount", "1");
```